tests/sqlparser_sqlite.rs

```diff
@@ -205,6 +205,17 @@ fn parse_create_sqlite_quote() {
     }
 }
 
+#[test]
+fn parse_create_table_gencol() {
+    let sql_default = "CREATE TABLE t1 (a INT, b INT GENERATED ALWAYS AS (a * 2))";
+    let sql_virt = "CREATE TABLE t1 (a INT, b INT GENERATED ALWAYS AS (a * 2) VIRTUAL)";
+    sqlite_and_generic().verified_stmt(sql_virt);
```

Comment:
Doesn't one_statement_parsers_to also verify the statement? (as in I think `verified_stmt` is redundant here)

Reply:
That makes sense, thanks - I'll remove the `verified stmt`.